### PR TITLE
upgrade drash to v1.x

### PR DESCRIPTION
**Pull request by crookse**
**Description**

* Use the 1.x version instead of specific versions so we don't run into any further upgrade issues
* Adds master workflow so we can run tests in the CI

### CODE REVIEW

**Comment by crookse** [comment]
Yea I’ll add those in. Can’t right now though.

**Comment by crookse** [comment]
> Should we add the badge to the readme as well? otherwise LGTM

added the badges

File: README.md [hunk 1]
```diff
@@ -3,6 +3,20 @@
   <h1 align="center">Drash Middleware</h1>
 </p>
 <p align="center">A middleware library for <a href="https://github.com/drashland/deno-drash">Drash</a></p>
+<p align="center">
+  <a href="https://github.com/drashland/deno-drash/releases">
```
Comment (ebebbington) [inline README.md:7]: needs to point to the middleware repo ;)

File: README.md [hunk 2]
```diff
@@ -3,6 +3,20 @@
...
+    <img src="https://img.shields.io/github/release/drashland/deno-drash-middleware.svg?color=bright_green&label=latest">
+  </a>
+  <a href="https://github.com/drashland/deno-drash/actions">
```
Comment (ebebbington) [inline README.md:10]: same as above

**Pull request merged**